hass_nabucasa/const.py

```diff
@@ -30,6 +30,7 @@
         "voice_api_url": "https://voice-api.nabucasa.com",
         "thingtalk_url": "https://thingtalk-api.nabucasa.com",
         "acme_directory_server": "https://acme-v02.api.letsencrypt.org/directory",
+        "migrate_subscription_url": "https://accounts.nabucasa.com/migrate_paypal_agreement",
```

Comment:
Do we want to have 1 url per variable or do we want to move to having “accounts.nabucasa.com” in constant file and we have paths relative to that?

Reply:
I'm fine with both, but if we change that it should be a breaking change, to remove all paths from this and only do hosts in this dictionary.

Reply:
I think that would be good. It would only be a breaking change for people that have configured a staging server.

Reply:
That is true 👍

Reply:
https://github.com/NabuCasa/hass-nabucasa/pull/381